apps/server/src/core/error/error.loggable.ts

```diff
@@ -0,0 +1,50 @@
+import { ApiValidationError } from '@shared/common';
+import { ErrorLogMessage, ILoggable, ValidationErrorLogMessage } from '../logger/interfaces/loggable';
+import { ErrorUtils } from './utils/error.utils';
+
+export class ErrorLoggable implements ILoggable {
+	error: Error;
+
+	constructor(error: Error) {
+		this.error = error;
+	}
+
+	getLogMessage() {
+		let logMessage: ErrorLogMessage | ValidationErrorLogMessage = {
+			error: this.error,
+			stack: this.error.stack,
+			type: 'Unhandled or Unknown Error',
+		};
+
+		if (this.error instanceof ApiValidationError) {
+			logMessage = this.createLogMessageForValidationErrors(this.error);
+		} else if (ErrorUtils.isFeathersError(this.error)) {
+			logMessage.type = 'Feathers Error';
+		} else if (
+			ErrorUtils.isBusinessError(this.error) ||
+			(ErrorUtils.isNestHttpException(this.error) && this.error.getStatus() < 500)
+		) {
+			logMessage.type = 'Business Error';
+		} else if (ErrorUtils.isNestHttpException(this.error)) {
+			// IMO it is incorrect to classify all Nest HttpExceptions as technical errors, because they are often used as business errors,
```

Comment:
since all codes below 500 have been handled in the case above, there are only 5xx errors left (as long as noone uses a 6xx code...)

What you could do for completeness would be to explicitly filter for 5xx codes, and mark other errors as not handleable or something. But I think it does make sense to mark 5xx codes as "technical error"

Reply:
I think it would be correct to mark 5xx codes as "technical error" but that is not what currently happens. Also 4xx errors are marked. 

But I also wonder why we need this explicit categarization in "technical errors" and "business errors" at all. Why not look at the error codes directly?